File: Mahasena.c
#include<stdio.h>
int main()
{
    int t,e=0,o=0;
    scanf("%d",&t);
    while(t--)
    {
        int n;
        scanf("%d",&n);
        if(n%2==0)
        e++;
        else
        o++;
    }
    if(e>o)
    printf("READY FOR BATTLE");
    else
    printf("NOT READY");
}

File: Mahasena.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        int c=0,d=0;
        while(t-- >0)
        {
            int x=sc.nextInt();
            if(x%2==0)
            c++;
            else
            d++;
        }
        if(c>d)
        System.out.println("READY FOR BATTLE");
        else
        System.out.println("NOT READY");
    }
}